### PR TITLE
Update Netlify instructions

### LINKED ISSUE #1139: Extending Prism to new language is not working
Extending Prism to a new language is not working.
I followed the documentation instructions:
1. I updated prism_syntax_highlighting = true in the config.toml file. 
2. I downloaded prism 1.28 with a new language. 
3. I copied the new prism.js and prism.css to static/js and static/css respectively.
4. I ran the command hugo server 
and nothing happens. The old prism 1.21 version was not replaced by the new version. 

I run macOS on M1 MacBook Air.

### PR DESCRIPTION
**Pull request by LisaFC**
Fixes #1139

### CODE REVIEW

**Review by emckean — APPROVED** [review]

File: userguide/content/en/docs/Deployment/_index.md [hunk 1]
```diff
@@ -40,12 +40,16 @@ Then follow the instructions in [Host on Netlify](https://gohugo.io/hosting-and-
...
-   1. In the **Advanced build settings** section, click **New variable**.
-   1. Specify `HUGO_VERSION` as the **Key** for the new variable, and `0.73` or later as its **Value**.
-   1. In the **Advanced build settings** section, click **New variable** again.
-   1. Specify `GO_VERSION` as the **Key** for the new variable, and `1.18` or later as its **Value**.
+   1. Specify your **Build command**. The exact build command depends on how you have chosen to use Docsy:
+      * If you are using Docsy as a [Git submodule](/docs/get-started/other-options/#option-1-docsy-as-a-git-submodule), specify `cd themes/docsy && git submodule update -f --init && cd ../.. && hugo`. You need to specify this rather than just `hugo` so that Netlify can use the theme's submodules.
+      * If you are using Docsy as a [Hugo module](/docs/get-started/docsy-as-module/) or NPM package, you can just specify `hugo`.   
+   3. Click **Show advanced**.
+   4. In the **Advanced build settings** section, click **New variable**.
+   5. Specify `HUGO_VERSION` as the **Key** for the new variable, and `0.73` or later as its **Value**.
```
Comment (deining) [inline userguide/content/en/docs/Deployment/_index.md:48]: I'm a bit concerned with explicitly specifying `0.73` here. What about:

Specify `HUGO_VERSION` as the **Key** for the new variable, and set its **Value** to the latest version of hugo (minimum recommended version: `0.73`).

File: userguide/content/en/docs/Deployment/_index.md [hunk 2]
```diff
@@ -40,12 +40,16 @@ Then follow the instructions in [Host on Netlify](https://gohugo.io/hosting-and-
...
+   6. In the **Advanced build settings** section, click **New variable** again.
+   7. Specify `GO_VERSION` as the **Key** for the new variable, and `1.18` or later as its **Value**.
```
Comment (deining) [inline userguide/content/en/docs/Deployment/_index.md:50]: Same here:

Specify `GO_VERSION` as the **Key** for the new variable, and set its **Value** to the latest version of go (minimum recommended version: `1.13`).

File: userguide/content/en/docs/Deployment/_index.md (hunk 1 above)
Comment (LisaFC) [reply userguide/content/en/docs/Deployment/_index.md:48]: Good call, thanks!

Comment (LisaFC) [reply userguide/content/en/docs/Deployment/_index.md]: Done!

File: userguide/content/en/docs/Deployment/_index.md (hunk 2 above)
Comment (LisaFC) [reply userguide/content/en/docs/Deployment/_index.md]: Done!

**Pull request merged**